### PR TITLE
C malecki/issue 32

### PR DESCRIPTION
**Pull request by c-malecki**
## Related Issues

| https://github.com/codefordayton/floswhistle-pandemic-v2/issues/32 |
| ---- |

## Impacted Areas in Application

List general components of the application that this PR will affect:

- Dashboard layout and styling
- mappage directory components

### CODE REVIEW

File: src/containers/mappage/MapInfoNational.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+            <div className="MapInfo_Section">
+              <div className="color-light-gray">Shortages Reported</div>
+
+              <div className="color-dark-blue larger-text">
+                {allTotals.citedShortage}
+              </div>
+            </div>
+
+            <div className="MapInfo_Section">
```
Comment (mmazanec22) [inline src/containers/mappage/MapInfoNational.js:44]: I think we can refactor this section later to DRY it up-- maybe by making an ordered array with the data that goes in this section, that includes objects with accessor and display functions for each of these things

File: src/containers/mappage/Mapv2.js [hunk 2]
```diff
@@ -53,26 +51,22 @@ class Mapv2 extends Component {
...
-                  </button>
-                  <button onClick={zoomOut} className="Map_Button">
-                    <i className="fas fa-minus"></i>
-                  </button>
-                  {/* <button onClick={resetTransform} className="Map_Button">
-                    x
-                  </button> */}
-                </div>
-              </div>
+              <Mapv2OverLay
```
Comment (mmazanec22) [inline src/containers/mappage/Mapv2.js:65]: Yay, glad this got abstracted into a component

File: src/containers/mappage/parsingmethods/districtParsing.js [hunk 3]
```diff
@@ -1,148 +1,44 @@
...
-  if (selectedDateDistrictArr.length === 0) {
-    return "No Reports";
-  } else {
-    const shortages = selectedDateDistrictArr
-      .map(({ testingUnavailable }) => testingUnavailable)
-      .reduce((a, b) => a + b, 0);
-    return shortages;
-  }
+  const totals = filteredDistricts
+    .map(({ citedShortage, citedNoTesting, reports }) => {
```
Comment (mmazanec22) [inline src/containers/mappage/parsingmethods/districtParsing.js:6]: Why map over this?  Just to eliminate fields from the objects in the array?

File: src/components/select/StyledSelect.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ import { withStyles } from "@material-ui/core";
 
 export const StyledSelect = withStyles({
   root: {
-    minWidth: "300px",
+    maxWidth: "300px",
```
[the remark this answers is not in the document]
Comment (c-malecki) [reply src/components/select/StyledSelect.js:6]: withStyles is only related to the Mui components that will be phased out. Everything else is currently based in plain old CSS.

File: src/containers/mappage/parsingmethods/districtParsing.js (hunk 3 above)
Comment (c-malecki) [reply src/containers/mappage/parsingmethods/districtParsing.js:6]: **citedShortage, citedNotesting**, and **reports** are top level fields within the array objects. **shortages** and **testData** are deeper obects, so the map is to isolate the parts of the array  to reduce the properties within the deeper objects.

**Pull request merged**